cron/cron-lib.pl

```diff
@@ -1605,9 +1605,8 @@ sub cleanup_temp_files
 closedir(DIR);
 
 # Cleanup old websockets
-if (&foreign_check("xterm")) {
-	&foreign_require("xterm");
-	&xterm::cleanup_miniserv_websockets();
+foreach (&get_miniserv_websockets_modules) {
```

Comment:
Shouldn't this be `&get_miniserv_websockets_modules()`

Reply:
Oh, you're right! Fixed now! Thanks!